src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -81,7 +82,16 @@ private string GetResultsFilePath(AssemblyInfo assemblyInfo)
                 // an empty log just in case, so our runner will still fail.
                 File.Create(resultsFilePath).Close();
 
-                var dumpOutputFilePath = Path.Combine(resultsDir, $"{assemblyInfo.DisplayName}.dmp");
+                // Define environment variables for processes started via ProcessRunner.
+                var environmentVariables = new Dictionary<string, string>();
+                environmentVariables.Add(ProcDumpRunner.ProcDumpPathEnvironmentVariableKey, _options.ProcDumpPath);
```

Comment:
@jaredpar The challenge here was the runner needs to effectively pass the variable to the unit test itself. I don't think xunit supports some way to do that but would love to be proved wrong. (I suggested this to @ivanbasov and wasn't happy with myself either.)